diff --git a/fs/proc/meminfo.c b/fs/proc/meminfo.c
index 83be312159c9..95279525777e 100644
--- a/fs/proc/meminfo.c
+++ b/fs/proc/meminfo.c
@@ -41,6 +41,7 @@ static int meminfo_proc_show(struct seq_file *m, void *v)
 	unsigned long sreclaimable, sunreclaim;
 	int lru;
 
+	pr_info("[test] this is a dynamic patch in meminfo_proc_show()!");
 	si_meminfo(&i);
 	si_swapinfo(&i);
 	committed = vm_memory_committed();
